deny.toml

```diff
@@ -24,8 +23,33 @@ unknown-git = "allow"
 allow-git = []
 
 [licenses]
-unlicensed = "allow"
-allow-osi-fsf-free = "either"
-copyleft = "deny"
-allow = ["MPL-2.0"]
+private.ignore = true
+unused-allowed-license = "allow"
+# List of explicitly allowed licenses
+# These are explicitly allowed by Cloudflare.
```

Comment:
foundations is a general-purpose library that is also used outside Cloudflare and these licenses are not universally suitable. Can you revise the allowlist to match the original intent?

Reply:
I've made the switch and opted to only list the licences currently in use.
Cargo deny will let us know if a licence is osi or fsf-free so we can add new ones as needed.